crates/sui-core/Cargo.toml

```diff
@@ -82,7 +82,7 @@ sui-storage.workspace = true
 sui-types.workspace = true
 workspace-hack.workspace = true
 zeroize.workspace = true
-mysticeti-core = { git = "https://github.com/MystenLabs/mysticeti", rev = "0bc1ae8f6cc5b1eb22752fa37ba64bdaf358f7cf" }
+mysticeti-core = { git = "https://github.com/MystenLabs/mysticeti", rev = "2d773a98198a33768a173a6f420de48a7e0a1880" }
```

Comment:
This will be replaced with the commit once this has been merged https://github.com/MystenLabs/mysticeti/pull/36